src/pages/workspaces/workspace/Data.js

```diff
@@ -358,38 +373,38 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
         input({ type: 'hidden', name: 'attributeNames', value: _.map('name', _.filter('visible', columnSettings)).join(',') }),
         input({ type: 'hidden', name: 'model', value: 'flexible' })
       ]),
+      'Download TSV',
```

Comment:
This doesn't look quite right, is this how you intended it?
<img width="894" alt="screen shot 2018-11-29 at 2 38 56 pm" src="https://user-images.githubusercontent.com/2395211/49247153-999b8780-f3e4-11e8-8055-ed9aec266810.png">

Reply:
Nope. That's one thing I was looking for design help with. After chatting with Brett and Brad, this is going back to just 2 buttons, one for download and one for copy. The download button will be for downloading selected rows when anything is selected. When nothing is selected, it will download everything.

Also, the select all and clear all are going to be in a menu using a `PopupTrigger` on an arrow next to the checkbox. That way they won't conditionally displace the rest of the UI.

I'll have an updated PR soon.